### DISCUSSION TITLE
Would there be plan for supporting "large world coordinates" in bevy?

### DISCUSSION THREAD

**Discussion by ameaninglessname** [bevyengine/bevy#10505] [Ideas]
[Click here for more information](https://docs.unrealengine.com/latest/en-US/large-world-coordinates-in-unreal-engine-5/)

(Just want to raise this topic, I think it should be considered as early as possible, for support games that has much bigger size in the future)

**Comment by afonsolage** [comment]
There were some discussion in the past, but for now, there are some crates which could help with that: https://github.com/aevyrie/big_space.

If you wanna see the discord discussion: https://discord.com/channels/691052431525675048/692572690833473578/1170800765271539852, but TL;TD; at some point Bevy will have to do it, but don't know when.
